packages/material-ui/src/Slider/Slider.js

```diff
@@ -743,7 +743,6 @@ const Slider = React.forwardRef(function Slider(props, ref) {
         return (
           <ValueLabelComponent
             key={index}
-            valueLabelFormat={valueLabelFormat}
```

Comment:
Why was this changed? We should definitely add a test for the behavior intended when using this prop.

Reply:
Private prop

Reply:
We should keep it separate.